transport/src/main/java/io/netty/channel/WriteBufferWaterMark.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel;
+
+/**
+ * WriteBufferWaterMark is used to set low water mark (default value is
+ * {@value #DEFAULT_WRITEBUFFER_LOW_WATERMARK} bytes) and high water
+ * mark (default value is {@value #DEFAULT_WRITEBUFFER_HIGH_WATERMARK} bytes)
+ * for write buffer.
```

Comment:
This isn't a summary. Imagine what the Javadoc of this class will look like with such long first sentence. Please provide a one-line description that ends with a dot.